src/openassetio-python/tests/package/hostApi/test_manager.py

```diff
@@ -240,6 +241,54 @@ def test_when_entity_ref_prefix_type_invalid_then_debug_log_printed(
         )
 
 
+class Test_Manager_initizlize_capablility_check:
+    def test_has_capability_called_after_initialize(
+        self, manager, a_host_session, mock_manager_interface
+    ):
+        manager.initialize({})
```

Comment:
[question] Interesting that this passes without having to set up `hasCapability`'s `return_value`, is the default return being coerced to a `bool`?

Reply:
ooh yeah - it must be!